tests/Fixture.spec.js

```diff
@@ -0,0 +1,88 @@
+test('Fixture I', ({assert, done}) => {
+	var updated = false
+	var container = document.createElement('div')
+	
+	var C = function() {
+	  if (updated === false) {
+	    updated = true
+	    this.forceUpdate()
+	  }
+	  return h('div', 'child')
+	}
+
+	var setState = null
+	var updated2 = false
+	
+	var P = function() {
+	  if (updated2 === false) {
+	    updated2 = true
+	    this.forceUpdate()
+	  }
+	  setState = this.setState.bind(this)
+	  return h('p', h('form', 'x'), this.state.test ? C : [])
```

Comment:
replace with

```js
h('p', _.filter([h('form'), this.state.test ? C : void 0])
```

to again produce the error

Reply:
perhaps additional unit testing would help avoid this

Reply:
I think i was chasing a red herring, the problem seems to step from be the call to forceUpdate in `render`.